Mathlib/Lean/ContextInfo.lean

```diff
@@ -86,4 +86,13 @@ def runTacticCode (ctx : ContextInfo) (i : TacticInfo) (goal : MVarId) (code : S
   ctx.runTactic i goal fun goal =>
     Lean.Elab.runTactic' (ctx := termCtx) (s := termState) goal code
 
+/-- Run tactic code, given by a piece of syntax, in the context of an infotree node. -/
+def runTacticCodeWithTypes (ctx : ContextInfo) (i : TacticInfo) (goal : MVarId) (code : Syntax) :
+    CommandElabM (List (MVarId × Expr)) := do
```

Comment:
Is the `MVarId` in the output ever actually used?  I wonder if you could simply return the expression of the goals.

Reply:
Thanks, done!